ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/TestKsqlRestApp.java

```diff
@@ -403,7 +408,8 @@ private static KsqlRestConfig buildConfig(
     final HashMap<String, Object> config = new HashMap<>(baseConfig);
 
     config.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, bootstrapServers.get());
-    config.putIfAbsent(KsqlRestConfig.LISTENERS_CONFIG, "http://localhost:0,https://localhost:0");
```

Comment:
How come the HTTPS listener was removed from here?

Reply:
It doesn't appear to be used (the SSL tests override this config), and also the new server doesn't support having an https listener if you haven't configured a key store. The old server did seem to support this, which seems a bit weird (how can you server https without having a server certificate?)